app/models/following.rb

```diff
@@ -1,8 +1,13 @@
 class Following
-  attr_reader :login, :html_url
+  attr_reader :login, :html_url, :github_id
 
   def initialize(attributes)
     @login = attributes[:login]
     @html_url = attributes[:html_url]
+    @github_id = attributes[:id]
+  end
+
+  def find_by_id
+    User.exists?(github_id: github_id)
```

Comment:
Make method to see if a user is in the DB or not so either show the Add Friends button or not